harness/determined/core/_train.py

```diff
@@ -5,8 +5,9 @@
 
 import determined as det
 from determined import tensorboard
-from determined.common import api
-from determined.common.api import errors
+from determined.common import api, util
+from determined.common.api import bindings, errors
+from determined.common.util import LEGACY_TRAINING, LEGACY_VALIDATION, Metrics, MetricType
```

Comment:
per the [google style guide](https://google.github.io/styleguide/pyguide.html#22-imports), please import packages, not individual classes/variables